docs/Project-Bootup-Flow.md

```diff
@@ -0,0 +1,24 @@
+# Project Boot-Up Flow
+
+These graphs show visually the flow of the bootup process of the project once the client runs the `vagrant up` command.
+
+- ## Boot-Up Structure
+    ```mermaid
+    flowchart TB
+    subgraph Vagrant["Vagrant (With 'fedora/34-cloud-base' Running On VirtualBox)"]
```

Comment:
> Running On VirtualBox

Consider this Virtual Machine running in VirtualBox

Reply:
This was updated in [aa55a5e](https://github.com/redhat-beyond/JobSeeker/commit/aa55a5e1b3ca36d88ae12d308465cf129dc85464). You may review again.